src/Chapter14/Listing14.10.HandlingExceptionsFromSubscribers.cs

```diff
@@ -19,9 +17,11 @@ public float CurrentTemperature
                 if(value != CurrentTemperature)
                 {
                     _CurrentTemperature = value;
-                    Action<float>? onTemperatureChange = OnTemperatureChange;
+                    Action<float>? 
+                        onTemperatureChange = OnTemperatureChange;
```

Comment:
I think we want this to all be on one line

Reply:
If we do need to split this across lines I would do it at the `=`
```C#
Action<float>? onTemperatureChange
    = OnTemperatureChange;
```